.github/workflows/build_test.yml

```diff
@@ -0,0 +1,55 @@
+name: Build/Test for PR and collaborator push
+
+on:
+  # allows us to run workflows manually
+  workflow_dispatch:
+  pull_request:
+    paths-ignore:
+      - '.github/workflows/build_test_publish.yml'
+      - 'docker/**'
+      - 'doc/**'
+  push:
+    paths-ignore:
+      - '.github/workflows/build_test_publish.yml'
+      - 'docker/**'
+      - 'doc/**'
+
+jobs:
+  build-and-test:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        ubuntu_versions : [
+          20.04,
+          22.04,
+        ]
+        pkg_mgr : [
+          apt,
+          conda,
+        ]
+        cyclus_tag: [
+          latest,
+        ]
+    
+    container:
+      image: ghcr.io/cyclus/cyclus_${{ matrix.ubuntu_versions }}_${{ matrix.pkg_mgr }}/cyclus:${{matrix.cyclus_tag}}
+
+    steps:
+      - name: Checkout Cycamore
+        uses: actions/checkout@v3
+
+      - name: Build Cycamore
+        run: |
+          python install.py --prefix=/root/.local --cyclus-root=/root/.local -j 2 --build-type=Release --core-version 99999.99999
+
+      - name: Cycamore Unit Tests
+        run: |
+          cycamore_unit_tests
+
+      - name: Cycamore Python Tests
+        run: |
+          export PYTHONPATH=$(find /root/.local/lib -type d -name 'cyclus-*' -print -quit)

```

Comment:
Curious what directory name we're looking for? I assume we can't be any less ambiguous than `cyclus-*`?

Reply:
Looking for the cyclus python installation directory `cyclus-<cyclus version>-<python version>.egg`.  e.g in the case of (apt, 20.04) this is `cyclus-999999.999999-py3.8.egg` but in (apt, 22.04) it is `cyclus-999999.999999-py3.10.egg`, etc.  

This was the alternative I found to setting `HOME` as `/root` so that python can find the cyclus module.  I can make it less ambiguous: `cyclus-*-*.egg`